# CRISPR_Zebrafish_ZEG

This project is intended as a supplement to the paper "Improved selection of zebrafish CRISPR editing by early next-generation sequencing based genotyping".

The input is a datafile generated by the BATCH-GE script (Boel A, Steyaert W et al,  Sci Rep 6, 30330 (2016)), containing the counts of individual insertion and deletion sequences for each sample, designated here as "Variants.INS.Seq.annotated". The naming of the samples is structured in the following order:
"KI", "date of experiment (numbers)", "ZEG or WE", "individual sample identifier".

The file is restructured via the following awk command lines:
awk -F"\t" '{print $3$4$5 "\t" $6 "\t" $8}' Variants.INS.Seq.annotated.txt > Var.txt
grep KI Variants.INS.Seq.annotated.txt  > test.txt
cat test.txt | awk '{print $3}' > names.txt

The Var.txt and names.txt are converted in to CSV files and used for the VarName.py script. The output CSV file, "fdf.csv" is used in downstream analysis. 
